### PR TITLE
feat: change loader by placeholder in List

### PR DESCRIPTION
**Pull request by Lawndlwd**
## Type

- Feature

Replace classic loader with the list placeholder

### CODE REVIEW

**Comment by Lawndlwd** [comment]
TODO: add default perPage

File: src/components/List/index.tsx [hunk 1]
```diff
@@ -87,20 +80,13 @@ function Body<DataType extends Record<string, unknown>>({
     customLoader,
     isLoading,
     emptyListComponent,
-    perPage,
+    perPage = 2,
```
Comment (matthprost) [inline src/components/List/index.tsx:83]: Why?

**Review by baptisteArno — CHANGES_REQUESTED** [review]

**Review by matthprost — APPROVED** [review]

**Comment by baptisteArno** [comment]
That still doesn't feel right IMO

CleanShot 2022-09-01 at 15 34 03@2x

Here I would expect 3 lines per row (none for the checkbox column) and each line would have the width of the row

**Comment by Lawndlwd** [comment]
> Here I would expect 3 lines per row (none for the checkbox column) and each line would have the width of the row

the 4th line not for the checkbox, it is for unlabelled col that contain action,
yeah, I tried to get style from the list it so complicated, i prefer to go with this for now @baptisteArno

**Review by philibea — APPROVED** [review]

**Review by baptisteArno — APPROVED** [review]

**Pull request merged**